vllm/utils.py

```diff
@@ -507,7 +506,13 @@ def create_kv_caches_with_random_flash(
         key_value_cache = torch.empty(size=key_value_cache_shape,
                                       dtype=torch_dtype,
                                       device=device)
-        key_value_cache.uniform_(-scale, scale)
+        if cache_dtype in ["auto", "half", "bfloat16", "float"]:
```

Comment:
is there no case "auto" becomes fp8?

Reply:
besides, handling auto here is a little ugly.. (feel like the caller should convert it alrady) but don't need to handle it here

Reply:
I am just following the pattern laid out in the non-flash function

Reply:
"auto" now means following the model weights/checkpoints. We should definitely improve the logic in the future.